docs/source/docker/getting_started/first_steps.rst

```diff
@@ -148,26 +148,24 @@ For example, if this is your input directory:
     L some/subfolder/
         L my-third-video.mp4
 
-And you put the following **filenames.txt** in the shared directory:
+Then you can specify two input files by creating the following **filenames.txt**:
 
 .. code-block:: console
 
     my-video.mp4
     some/subfolder/my-third-video.mp4
 
-Then you can use the following docker run command and the Lightly docker will only consider **my-video.mp4** and **my-third-video.mp4**.
-
-.. code-block:: console
-
-    docker run --gpus all --rm -it \
-        -v {INPUT_DIR}:/home/input_dir:ro \
-        -v {SHARED_DIR}:/home/shared_dir \
-        -v {OUTPUT_DIR}:/home/output_dir \
-        lightly/worker:latest \
-        token=MYAWESOMETOKEN \
-        relevant_filenames_file='filenames.txt'
+If you use a cloud bucket as input datasource, upload the file to it
+and copy the path of the file relative to the datasource root.
+If you use a cloud bucket and specified a separate input and output bucket,
+put the file in the .lightly folder of the output bucket
+and copy the path of the file relative to the output datasource root.
```

Comment:
I think a small example with the full path to the relevant filenames txt and the path you have to specify in the config would be nice.